docusaurus/docs/develop/path/cheat_sheet_shannon.md

```diff
@@ -0,0 +1,257 @@
+
+```bash
+poktrolld tx gateway stake-gateway \
+ --config=/tmp/stake_gateway_config.yaml \
+ --from=gateway --gas=auto --gas-prices=1upokt --gas-adjustment=1.5 --chain-id=pocket-beta \
+ --node=https://shannon-testnet-grove-rpc.beta.poktroll.com \
+ --yes
+```
+
+**Stake the application:**
+
+```bash
+poktrolld tx application stake-application \
+ --config=/tmp/stake_app_config.yaml \
+ --from=application --gas=auto --gas-prices=1upokt --gas-adjustment=1.5 --chain-id=pocket-beta \
+ --node=https://shannon-testnet-grove-rpc.beta.poktroll.com \
+ --yes
+```
+
+**Delegate from the application to the gateway:**
+
+```bash
+poktrolld tx application delegate-to-gateway $(poktrolld keys show -a gateway) \
+ --from=application --gas=auto --gas-prices=1upokt --gas-adjustment=1.5 --chain-id=pocket-beta \
+ --node=https://shannon-testnet-grove-rpc.beta.poktroll.com \
+ --yes
+```
+
+</details>
+
+### 1.2 `Application` and `Gateway` Account Validation
+
+After following the instructions above, you should have keys with the names `gateway` and `application` in your keyring.
+
+You can validate it like so:
+
+```bash
+# All accounts
+poktrolld keys list
+
+# Gateway only
+pkd keys show -a gateway
+
+# Application only
+pkd keys show -a application
+```
+
+## 2. Configure PATH for Shannon
+
+### 2.1 Generate Shannon Config
+
+Run the following command to generate a Shannon config at `local/path/config/.config.yaml`:
+
+```bash
+make shannon_populate_config
+```
+
+Note that running `make shannon_populate_config` is equivalent to running the following commands:
+
+```bash
+make prepare_morse_e2e_config # Generate ./e2e/.shannon.config.yaml
+make copy_morse_e2e_config_to_local # Copy to ./local/path/config/.config.yaml
+```
+
+:::warning Private Key Export
+
+1. **Ignore instructions** that prompt you to update the file manually.
+2. **Select `y`** to export the private keys for the script to work
+
+:::
+
+### 2.2 Verify Configuration
+
+Check your config file:
+
+```bash
+cat local/path/config/.config.yaml
+```
+
+It should look similar to the following with the `gateway_config` filled out.
+
+```yaml
+shannon_config:
+  full_node_config:
+    rpc_url: https://shannon-testnet-grove-rpc.beta.poktroll.com
+    grpc_config:
+      host_port: shannon-testnet-grove-grpc.beta.poktroll.com:443
+    lazy_mode: true
+
+  gateway_config:
+    gateway_mode: "centralized"
+    gateway_address: pokt1... # Your gateway address
+    gateway_private_key_hex: "0x..." # Your gateway private key
+    owned_apps_private_keys_hex:
+      - "0x..." # Your application private key
+hydrator_config:
+  service_ids:
+    - "anvil"
+auth_server_config:
+  grpc_host_port: path-auth-data-server:50051
+  grpc_use_insecure_credentials: true
+  endpoint_id_extractor_type: url_path
+```
+
+:::important Gateway Configuration
+Ensure that `gateway_config` is filled out correctly before continuing.
+:::
+
+## 3. Start PATH
+
+Run the entire stack (PATH, Envoy, Auth Server) by running:
```

Comment:
Same comment as above re: needing to prepare/populate the Envoy config files. I see the Walkthrough section includes links to the Tilt setup page but IMO it would be valuable to link to it from here and from the Morse cheat sheet as well.

Reply:
Going to reply in one "mega comment"